.gitlab/functional_test/regression_detector.yml

```diff
@@ -17,7 +17,7 @@ single-machine-performance-regression_detector:
       - outputs/report.html # for debugging, also on S3
     when: always
   variables:
-    SMP_VERSION: 0.15.1
+    SMP_VERSION: 0.16.0
```

Comment:
How about `0.16.1`?

Reply:
it never got published, at this point I'm just waiting for 0.17